packages/desktop-client/src/browser-preload.browser.js

```diff
@@ -18,11 +18,16 @@ function createBackendWorker() {
   worker = new BackendWorker();
   initSQLBackend(worker);
 
+  if (window.SharedArrayBuffer) {
+    localStorage.removeItem('SharedArrayBufferOverride');
+  }
+
   worker.postMessage({
     type: 'init',
     version: ACTUAL_VERSION,
     isDev: IS_DEV,
-    hash: process.env.REACT_APP_BACKEND_WORKER_HASH
+    hash: process.env.REACT_APP_BACKEND_WORKER_HASH,
+    allowBuggyFallback: localStorage.getItem('SharedArrayBufferOverride')
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I'm OK with the proposed implementation, but could we name the variable more concisely? "Buggy" is very non-descriptive.

Here's my suggestion, but I'm open to other ideas too:

```suggestion
    isSharedArrayBufferOverrideEnabled: localStorage.getItem('SharedArrayBufferOverride')
```